ale_linters/llvm/llc.vim

```diff
@@ -0,0 +1,35 @@
+" Author: rhysd <https://rhysd.github.io>
+" Description: Support for checking LLVM IR with llc
+
+call ale#Set('llvm_llc_executable', 'llc')
+
+function! ale_linters#llvm#llc#GetExecutable(buffer) abort
+    return ale#Escape(ale#Var(a:buffer, 'llvm_llc_executable'))
```

Comment:
Nope, don't escape this here. The return value of the callbacks for getting the executable must not be a escaped, but the executable should be escaped in the command callback.

Reply:
Ah, I got it.